counter.go

```diff
@@ -59,16 +59,14 @@ func (c *Counter) Add(delta int64) {
 	t, ok := ptokenPool.Get().(*ptoken)
 	if !ok {
 		t = new(ptoken)
-		t.idx = fastrand() & c.mask
 	}
+	t.idx = fastrand() & c.mask
```

Comment:
Thank you so much for the spotting the bug! However, this fix kills the whole idea of self-balancing goroutine assignment to stripes. I've created #74 with a fix that keeps the algorithm as is.